Source/WebKit/UIProcess/glib/DisplayVBlankMonitorDRM.cpp

```diff
@@ -176,9 +191,32 @@ static int crtcBitmaskForIndex(uint32_t crtcIndex)
 
 std::unique_ptr<DisplayVBlankMonitor> DisplayVBlankMonitorDRM::create(PlatformDisplayID displayID)
 {
+#if PLATFORM(WPE) && ENABLE(WPE_PLATFORM)
+    bool usingWPEPlatformAPI = !!g_type_class_peek(WPE_TYPE_DISPLAY);
```

Comment:
This could be `static`.

```suggestion
    static bool usingWPEPlatformAPI = !!g_type_class_peek(WPE_TYPE_DISPLAY);
```